test/e2e/prometheus_test.go

```diff
@@ -3640,19 +3640,21 @@ func testPromWebTLS(t *testing.T) {
 
 	prom := framework.MakeBasicPrometheus(ns, "basic-prometheus", "test-group", 1)
 	prom.Spec.Web = &monitoringv1.PrometheusWebSpec{
-		TLSConfig: &monitoringv1.WebTLSConfig{
```

Comment:
I would extend the test to also cover `http_server_config`.

Reply:
@simonpasquier may I know how would you extend the test? IMHO, we already have an unit test in `webconfig` pkg to cover the `http_server_config` config generation and the `http_server_config` functionality is promised by Prometheus and Alertmanager if user write a correct config. This is why I didn’t write any e2e test case for this part.

Reply:
it will help detecting potential regressions in the future and since we have an existing test, it should be a reasonable amount of work :)